omdb-client/api/src/main/java/com/garret/movies/omdb/dto/MoviesResponse.java

```diff
@@ -0,0 +1,16 @@
+package com.garret.movies.omdb.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class MoviesResponse {
+    @JsonProperty("Search")
+    private List<ShortMovie> shortMovieList;
+    @JsonProperty("totalResults")
+    private String totalResults;
+    @JsonProperty("Response")
+    private String response;
```

Comment:
What value we get here?

Reply:
@mmarusich true or false but in string type